Formula/proxysql.rb

```diff
@@ -0,0 +1,45 @@
+class Proxysql < Formula
+  desc "General-purpose data compression with high compression ratio"
+  homepage "http://www.proxysql.com/"
+  url "https://github.com/sysown/proxysql/archive/v1.4.9.tar.gz"
+  # mirror "https://tukaani.org/xz/xz-5.2.4.tar.gz"
+  sha256 "28ee75735716ab2e882b377466f37f5836ce108cfcfe4cf36f31574f81cce401"
+
+  # Build dependencies listed here: https://github.com/sysown/proxysql/blob/master/INSTALL.md
+  depends_on "automake" => :build
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+  depends_on "curl"
+  depends_on "openssl"
+
+  # https://github.com/sysown/proxysql/pull/1564
+  # TODO when the 1.4.10 release is cut, this formula should be updated and
+  # this patch should be removed.
+  patch do
+    url "https://github.com/sysown/proxysql/compare/v1.4.10...zbentley:v1.4.10.diff?full_index=1"
```

Comment:
This needs to be accepted by upstream first and we'll probably want to wait for a new release so the patches aren't needed.

Reply:
Would a compromise be acceptable here? If upstream accepts that PR but does not cut a release, could we ship the formula backporting the change until the release is cut? I promise that, unless I lose my job and access to a computer, I'll update the formula as soon as .10 is cut. 

I have a moderate number of users for which installing a custom formula will be confusing. That's the only reason I ask this.

Reply:
Patches accepted.